.gitignore

```diff
@@ -5,4 +5,7 @@ compiled
 
 include/document_graph
 include/logger
-src/document_graph
\ No newline at end of file
+src/document_graph
+test/probs.js
```

Comment:
you can put just test/* to ignore all test files

Reply:
Also remove the test file, (at least don't add / push files that are not related or used to the project)